bin/gbuild

```diff
@@ -159,9 +159,13 @@ EOF" if build_desc["sudo"] and @options[:allow_sudo]
     script.puts "cd build"
     script.puts build_desc["script"]
   end
-
+  
```

Comment:
Why this change?

Reply:
Well the moment, you don't have a real code editor handling the whitespace situation for you 💃 
Good find tho